src/lib.rs

```diff
@@ -10,11 +10,14 @@ pub mod arg;
 pub mod azidentityext;
 pub mod client;
 pub mod cmd;
+pub mod log;
 
 #[cfg(target_arch = "wasm32")]
 pub mod wasm_exports;
 
 pub async fn run(p: PathBuf, client: &Client, raw_input: Vec<String>) -> Result<String> {
+    tracing::debug!("Running CLI with input: {:?}", raw_input);
```

Comment:
Is there a way to set the debug level so that these debug information are only displayed when with `--debug`?